docker-compose.yml

```
@@ -0,0 +1,9 @@
+services:
+  smartmeter:
+    build: .
+    container_name: smartmeter
+    restart: unless-stopped
+    devices:
+      - ${SMARTMETER_SERIAL_DEVICE}:/dev/ttyUSB0
+    volumes:
+      - ${SMARTMETER_DATACOLLECTOR_INI}:/app/datacollector.ini:rw
```

```suggestion
      - ${SMARTMETER_DATACOLLECTOR_INI}:/app/datacollector.ini:ro
```
since the `smartmeter_datacollector` application does not modify the configuration file.